### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#3534] [Releases & Announcements]
## Changelog

We are happy to announce 2.0 - a new major version of Chatwoot that improves product stability, adds new features and productivity enhancements, and a complete design overhaul. This release also marks our first Enterprise edition offering.

- IMAP and SMTP support for email channels 
- Dedicated View for conversation mentions
- Advanced conversation filters
- Advanced contact filters
- Template support for 360 dialog WhatsApp channels
- List and Checkbox support for custom attributes
- View conversation summary in notification emails
- OpenAPI compliant swagger docs
- Numerous bug fixes and enhancements

Thanks to @shivamsinghchahar @jdsalchow, @vinayak-gaikwad, @akhilgkrishnan, @SanjuSudheerM, @santhosh-chinnasamy @thedev105  for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by Brokray** [comment]
Amazing job guys, congratulations!  

As it is not mentioned, I guess the update process is the same as usual or is there any breaking change for self-hosted versions?    

Cheers

**Reply by vishnu-narayanan** [reply]
@Brokray Yes, same as usual. No breaking changes.
